gradle/spotbugs-exclude.xml

```diff
@@ -222,6 +222,12 @@ For a detailed description of spotbugs bug categories, see https://spotbugs.read
         <Bug pattern="MS_MUTABLE_ARRAY"/>
     </Match>
 
+    <Match>
+        <!-- The code generator generates useless condition. Disable the check temporarily. -->
+        <Class name="org.apache.kafka.common.message.JoinGroupResponseData"/>
+        <Bug pattern="UC_USELESS_CONDITION"/>
+    </Match>
+
```

Comment:
With the `nullable` Protocol Name, the code generator generates unreachable condition. I have added an exception for the time being and will address this in a follow up PR if you don't mind. It does not impact the behaviour of this PR.